### PR TITLE
LPS-63943

### PR DESCRIPTION
**Pull request by Ithildir**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
ci:stop

**Comment by brianchandotcom** [comment]
@codyhoag fyi

File: modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown [hunk 1]
```diff
@@ -1182,6 +1182,15 @@ their dependencies published before enabling the `printArtifactPublishCommands`
 task.
 - [LPS-71005]: Update the [Liferay Gradle Plugins] dependency to version 3.2.15.
 
+## 3.2.5 - 2017-03-08
+
+### Added
+- [LPS-63943]: Print out artifacts publish commands file path.
```
Comment (codyhoag) [inline modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown]: @Ithildir Can you describe this one a bit more? Would something like this be correct?

> Print out the artifacts' file paths when executing publish commands.

Comment (Ithildir) [inline modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown]: The `writeArtifactsPublishCommands` task creates a `.sh` file with a list of commands that Brian runs in order to publish modules. I added a log, so at the end of the task it writes where this `.sh` file is.

**Pull request closed without merging**